observability/otel/otel-collector/otel-collector-processors.adoc

```diff
@@ -574,6 +576,342 @@ config:
+            type: numeric_attribute,
+            numeric_attribute: {key: <key1>, min_value: 50, max_value: 100} # <1>
+          },
+        ]
+# ...
+----
+<1> The provided `50` and `100` values are examples.
+
+* The following policy samples only a percentage of traces:
++
+----
+# ...
+      policies:
+        [
+          {
+            name: <probabilistic_policy>,
+            type: probabilistic,
+            probabilistic: {sampling_percentage: 10} # <1>
+          },
+        ]
+# ...
+----
+<1> The provided `10` value is an example.
+
+* The following policy samples traces by the status code: `OK`, `ERROR` or `UNSET`:
++
+----
+# ...
+      policies:
+        [
+          {
+            name: <status_code_policy>,
+            type: status_code,
+            status_code: {status_codes: [ERROR, UNSET]}
+          },
+        ]
+# ...
+----
+
+* The following policy samples traces by string value matches for resource and record attributes:
++
+----
+# ...
+      policies:
+        [
+          {
+            name: <string_attribute_policy>,
+            type: string_attribute,
+            string_attribute: {key: <key2>, values: [<value1>, <val>*], enabled_regex_matching: true, cache_max_size: 10} # <1>
+          },
+        ]
+# ...
+----
+<1> This policy definition supports both exact and regular-expression value matches. The provided `10` value in the `cache_max_size` field is an example.
+
+* The following policy samples traces by the rate of spans per second:
++
+----
+# ...
+      policies:
+        [
+          {
+            name: <rate_limiting_policy>,
+            type: rate_limiting,
+            rate_limiting: {spans_per_second: 35} # <1>
+          },
+        ]
+# ...
+----
+<1> The provided `35` value is an example.
+
+* The following policy samples traces by the minimum and maximum number of spans inclusively:
++
+----
+# ...
+      policies:
+        [
+          {
+            name: <span_count_policy>,
+            type: span_count,
+            span_count: {min_spans: 2, max_spans: 20} # <1>
+          },
+        ]
+# ...
+----
+<1> If the sum of all spans in the trace is outside the range threshold, the trace will not be sampled. The provided `2` and `20` values are examples.
+
+* The following policy samples traces by `TraceState` value matches:
++
+----
+# ...
+      policies:
+        [
+          {
+            name: <trace_state_policy>,
+            type: trace_state,
+            trace_state: { key: <key3>, values: [<value1>, <value2>] }
+          },
+        ]
+# ...
+----
+
+* The following policy samples traces by a boolean attribute (resource and record):
++
+----
+# ...
+      policies:
+        [
+          {
+            name: <bool_attribute_policy>,
+            type: boolean_attribute,
+            boolean_attribute: {key: <key4>, value: true}
+          },
+        ]
+# ...
+----
+
+* The following policy samples traces by a given boolean OTTL condition for a span or span event:
```

Comment:
Does OTTL need to be defined?

Reply:
I would say this term is either known or discoverable to the target audience.
I considered defining it in-line here, but readability would suffer:
```given boolen OpenTelemetry Transformation Language (OTTL) condition```